modules/real_time_enforcer/main.tf

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  random_hash          = "${var.suffix}"
+  network_project      = "${var.network_project != "" ? var.network_project : var.project_id}"
+  enforcer_name        = "forseti-enforcer-vm-${local.random_hash}"
+  enforcer_sa_name     = "forseti-enforcer-gcp-${local.random_hash}"
+  enforcer_bucket_name = "forseti-enforcer-${local.random_hash}"
+  enforcer_zone        = "${var.enforcer_region}-c"
+
+  real_time_enforcer_policy_files = [
```

Comment:
Full list should be:
real_time_enforcer_policy_files = [
    "policy/policies.rego",
    "policy/exclusions.rego",
    "policy/config.yaml",
    "policy/bigquery/common.rego",
    "policy/bigquery/dataset_no_public_access.rego",
    "policy/bigquery/dataset_no_public_authenticated_access.rego",
    "policy/cloudresourcemanager/common_iam.rego",
    "policy/sql/acl.rego",
    "policy/sql/backups.rego",
    "policy/sql/common.rego",
    "policy/sql/require_ssl.rego",
    "policy/storage/bucket_iam_disallow_allauthenticatedusers.rego",
    "policy/storage/bucket_iam_disallow_allusers.rego",
    "policy/storage/common.rego",
    "policy/storage/common_iam.rego",
    "policy/storage/versioning.rego",
]

Reply:
Resolved in cc4e49c92be6aca4b6539fca8d1cfadaba2529ed.